### PR TITLE
update README style

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Maybe we should write something more in the README? Not sure how important it is since this is basically just a utility module used by `levelup`.

File: README.md [hunk 1]
```diff
@@ -1,26 +1,12 @@
-
 # level-errors
 
-<img alt="LevelDB Logo" height="100" src="http://leveldb.org/img/logo.svg">
-
-> Error module for [LevelUP](https://github.com/rvagg/node-levelup)
+> Error module for [levelup][levelup]
 
+[![level badge][level-badge]](https://github.com/level/awesome)
 [![Build Status](https://travis-ci.org/Level/errors.png)](https://travis-ci.org/Level/errors)
```
Comment (vweevers) [inline README.md]: Can you make the travis badge an SVG?

**Comment by vweevers** [comment]
> Not sure how important it is since this is basically just a utility module used by levelup.

Yeah, I even forgot this module *existed*.

File: README.md (hunk 1 above)
Comment (ralphtheninja) [reply README.md]: Sure!

**Review by vweevers — APPROVED** [review]

**Comment by ralphtheninja** [comment]
1.1.2

**Pull request merged**